app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivity.kt

```diff
@@ -26,13 +26,13 @@ const val SELECTED_PROFILE_ID_SAVED_KEY =
 /** Argument key for last loaded fragment in [AdministratorControlsActivity]. */
 const val LAST_LOADED_FRAGMENT_EXTRA_KEY = "AdministratorControlsActivity.last_loaded_fragment"
 
-/** Argument key for [ProfileListFragment]. */
+/** Argument key for identifying [ProfileListFragment] in the backstack. */
```

Comment:
Could we just say it's used to identify the fragment?

Also, should this be called a TAG per convention? And, is this the correct constant style for such constants? /cc @rt4914 since he'd be more familiar with these conventions.

Ditto for the others.